lib/fault_handler/Kconfig

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
maybe rename to fatal.c? or fatal_error.c?

Reply:
It can be fatal_error.c